src/app/record/select-skills/page.tsx

```
@@ -253,7 +240,7 @@ const Page = () => {
           createPortal(
             <div
               data-skill-select-menu
-              className="fixed z-[100] overflow-y-auto [scrollbar-width:none] [&::-webkit-scrollbar]:hidden"
+              className="fixed z-100 overflow-y-auto [scrollbar-width:none] [&::-webkit-scrollbar]:hidden"
```

ESLint

[error] Replace `overflow-y-auto·[scrollbar-width:none]` with `[scrollbar-width:none]·overflow-y-auto`

(prettier/prettier)